mungegithub/mungers/approvers/owners.go

```diff
@@ -308,19 +346,31 @@ func (ap Approvers) GetFiles() []File {
 	return allOwnersFiles
 }
 
+// GetCCs gets the list of suggested approvers for a pull-request.  It
+// now considers current assignees as potential approvers. Here is how
+// it works:
+// - We find suggested approvers from all potential approvers, but
+// remove those that are not useful considering current approvers and
+// assignees.
+// - We find a subset of suggested approvers from from current
+// approvers, suggested approvers and assignees, but we remove thoses
+// that are not useful considering suggestd approvers and current
+// approvers.
+// We return the union of the two sets: suggested and suggested
+// assignees.
+// The goal of this second step is to only keep the assignees that are
+// the most useful.
 func (ap Approvers) GetCCs() []string {
-	approvers := []string{}
+	randomizedApprovers := ap.owners.GetShuffledApprovers()
```

Comment:
Read it again: `ap` is the equivalent of `this`, `owners` is a private member of `ap`. `ap` can access its own private fields. What would be bad would be to do `ap.owners.repo.Something`